rollup.config.js

```diff
@@ -1,3 +1,9 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-env es6 */
+
+
+const babel = require('rollup-plugin-babel');
+const cleanup = require('rollup-plugin-cleanup');
```

Comment:
this was used to remove typescript comments from the main project. I don't think we need it here since this doesn't have any typescript comments

Reply:
When comparing the build output with and without, it does look like it cleans up the `// DEBUG` and 
```
	/**
	 * @private
	 */
```
code comments. but nothing major.

Reply:
This is only applied to the non-minified ES6 build. I think comments are helpful in the non-minified build for development in case you inspect source. In production, people will mostly use the minified build or more likely will use a bundler like webpack that will handle stripping out the comments. So I'd still prefer not adding it here

Reply:
Agreed, all good! I've removed it locally, just doing some due diligence at the moment and backing out the package.json additions and retesting